wiki/Tournaments/TEMPLATE.md

```diff
@@ -0,0 +1,142 @@
+---
+tags:
+  - {SHORT_TITLE}
+  - {SERIES_SHORT_TITLE}
+---
+
+# {TITLE}
+
+![{SHORT_TITLE} logo/banner](img/logo.png) <!-- choose the option that suits the image better -->
+
+The **{TITLE}** (***{SHORT_TITLE}***) is a <!-- enter applicable tournament traits, example: {N}v{N}, double-elimination, country-based --> {MODE} tournament hosted by the {ORGANISERS} <!-- list the organisers here with their flags -->. It is the {ITERATION_ORDINAL} installment of the {SERIES_TITLE}.
```

Comment:
i don't think anyone has kept the "the" before the person(s) who hosted the tournament, but let's remove it to prevent such a silly mistake

also british spelling (may want to do a search and replace on previous articles in a separate pr)

```suggestion
The **{TITLE}** (***{SHORT_TITLE}***) is a <!-- enter applicable tournament traits, example: {N}v{N}, double-elimination, country-based --> {MODE} tournament hosted by {ORGANISERS} <!-- list the organisers here with their flags -->. It is the {ITERATION_ORDINAL} instalment of the {SERIES_TITLE}.
```